src/style.css

```
@@ -1,3 +1,72 @@
+:root {
+  --main-textcolor: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo__image {
+  display: block;
+  height: 40px;
+  width: 40px;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: var(--main-textcolor);
+}
+
+.is-active {
+  color: var(--main-textcolor) ;
+}
```

You can combine selectors with the same styles
```suggestion
.nav__link:hover,
.is-active {
  color: var(--main-textcolor) ;
}
```